meeting_minutes/220714_SARIF_TC_62.md

```diff
@@ -0,0 +1,157 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair David)
+
+## 1.2 Introduction of participants/roll call (Co-Chair David)
+
+| First Name | Last Name | Company           | Role(s)                 |
+|:-----------|:----------|:------------------|:------------------------|
+| Aditya     | Sharad    | Microsoft         | Member                  |
+| Aharon     | Abadi     | WhiteSource       | Voting Member           |
+| Chris      | Meyer     | Microsoft         | Voting Member           |
+| Chris      | Wysopal   | Veracode          | Member                  |
+| David      | Keaton    | Individual        | Chair                   |
+| Jeff       | Williams  | Contrast Security | Voting Member           |
+| Johnathan  | Gilday    | Contrast Security | Member                  |
+| Mary       | Martin    | Microsoft         | Member                  |
+| Michael    | Fanning   | Microsoft         | Voting member           |
+| Paul       | Anderson  | Grammatech, Inc.  | Voting member           |
+| Pavel      | Mikula    | Sonar             | Member                  |
```

Comment:
I think I'm a voting member as I got voting rights last time.
```suggestion
| Pavel      | Mikula    | Sonar             | Voting member |
```

Reply:
Correct, thanks for catching the cut-and-paste errors.